wallet/wallet.go

```diff
@@ -3438,19 +3446,19 @@ func (w *Wallet) publishTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
 	// confirmed store. We'll avoid returning an error as the broadcast was
 	// in a sense successful.
 	//
-	// This error is returned when broadcasting/sending a transaction that
-	// has already confirmed to a btcd node.
-	case strings.Contains(err.Error(), "transaction already exists"):
+	// This error is returned when sending a transaction that has already
+	// confirmed to a btcd/bitcoind node over RPC.
+	case rpcTxConfirmed:
 		fallthrough
 
 	// This error is returned when broadcasting a transaction that has
-	// already confirmed to a bitcoind node.
-	case strings.Contains(err.Error(), "txn-already-known"):
+	// already confirmed to a btcd node over the P2P network.
+	case strings.Contains(err.Error(), "transaction already exists"):
```

Comment:
would prefer if both of these cases were made case insensitive for robustness

Reply:
Done.